app/geodiffutils.cpp

```diff
@@ -109,3 +109,17 @@ bool GeodiffUtils::applyDiffs( const QString &src, const QStringList &diffFiles
   }
   return true;
 }
+
+void GeodiffUtils::log( GEODIFF_LoggerLevel level, const char *msg )
+{
+  QString prefix;
+  switch ( level )
+  {
+    case LevelError: prefix = "GEODIFF error: "; break;
+    case LevelWarning: prefix = "GEODIFF warning: "; break;
+    case LevelInfo: prefix = "GEODIFF info: "; break;
+    case LevelDebug: prefix = "GEODIFF debug: "; break;
```

Comment:
Other prefixes used in InputUtils::log() do not end up with colon+space, so it would be good to remove them here for consistency